README.md

```diff
@@ -1,57 +1,11 @@
-# Northcoders House of Games API
+## How to use this repo
 
-## Background
+If you are using this repo, you will need to include a ".env.test" file, which points towards the test dataset using PGDATABASE = "Blah"
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+You will also need a ".env.development" file, using PGDATABASE = "Blah blah"
```

Comment:
Please sort this with the database name and what it needs to actually look like inside the file.